crates/symbolicator-service/src/caching/mod.rs

```diff
@@ -294,7 +295,10 @@ impl Caches {
                 config,
                 config.caches.downloaded.into(),
                 max_lazy_redownloads,
-                default_cap,
+                // We use 2GiB as the in-memory cache size for Proguard files.
+                // Note that a Proguard mapper can take up hundreds of MB
+                // in memory.
+                2 * 1024 * 1024 * 1024,
```

Comment:
some of the other caches are configurable using the `in_memory` config. can you move this default to over there?

(tbh, changing the config file is even more of a hassle than to just deploy a code change, as it would go through the `ops` repo. something we might want to bring up to a wider audience, though its not painful enough to be unbearable :-D)